Taskfile.yml

```diff
@@ -69,3 +69,11 @@ tasks:
     desc: Format all supported files with Prettier
     cmds:
       - npx prettier --write .
+
```

Comment:
```suggestion

  # Source: https://github.com/arduino/tooling-project-assets/blob/main/workflow-templates/assets/poetry-task/Taskfile.yml
  poetry:install-deps:
    desc: Install dependencies managed by Poetry
    cmds:
      - poetry install --no-root

```

Reply:
@silvanocerza. This suggestion was reverted by the force push:
https://github.com/arduino/serial-discovery/compare/f5dd033b4b82cbea4a16b19ad1446c6e093b0d98..2f1ee447302c59f12417ff01ca190c867a06d3b0#diff-cd2d359855d0301ce190f1ec3b4c572ea690c83747f6df61c9340720e3d2425eL74

Reply:
Sorry, will fix right away. :facepalm: